<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>581</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Lista de estudiantes</string>
   </property>
   <widget class="QTableWidget" name="lista">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>561</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>170</y>
     <width>141</width>
     <height>113</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Aporte 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Aporte1">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>0.00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Aporte 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Aporte2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>0.00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Promedio</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Promedio">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>0.00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>300</y>
     <width>77</width>
     <height>112</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btnnuevo">
      <property name="text">
       <string>Nuevo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnanadir">
      <property name="text">
       <string>Añadir</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnEliminar">
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnsalir">
      <property name="text">
       <string>Salir</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="fecha">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fecha</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>556</width>
     <height>53</height>
    </rect>
   </property>
   <property name="title">
    <string>Estudiante</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Apellido</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="apellido"/>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Nombre</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nombre"/>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Cedula</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="cedula"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>501</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Curso</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>20</y>
      <width>409</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Materia</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="materia"/>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Codigo de Materia</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="codigoMateria"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>450</y>
     <width>177</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QPushButton" name="GraficoBarras">
      <property name="text">
       <string>Gráfico de Barras </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ArbolBinario">
      <property name="text">
       <string>Árbol binario</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
